Userland/Libraries/LibSQL/AST/CreateSchema.cpp

```diff
@@ -12,7 +12,10 @@ namespace SQL::AST {
 
 RefPtr<SQLResult> CreateSchema::execute(ExecutionContext& context) const
 {
-    auto schema_def = context.database->get_schema(m_schema_name);
+    auto schema_def_or_error = context.database->get_schema(m_schema_name);
+    if (schema_def_or_error.is_error())
+        return SQLResult::construct(SQLCommand::Create, SQLErrorCode::InternalError, schema_def_or_error.error());
+    auto schema_def = schema_def_or_error.value();
```

Comment:
All of the `*_or_error.value()` calls here could be `*_or_error.release_value()` to avoid a `RefPtr` ref/deref.